sponsors/models.py

```diff
@@ -260,6 +263,9 @@ class Sponsorship(models.Model):
     level_name = models.CharField(max_length=64, default="")
     sponsorship_fee = models.PositiveIntegerField(null=True, blank=True)
 
+    def __str__(self):
+        return f"{self.level_name} ({self.get_status_display()}) for sponsor {self.sponsor.name}"
```

Comment:
may want to include start/end date in the string representation, it won't be an issue for first term (initial launch) but will be important for differentiating down the line.

Reply:
Fixed in d134d3c